cmd/supernode/app/root.go

```diff
@@ -175,11 +176,10 @@ func runSuperNode() error {
 // initLog initializes log Level and log format.
 func initLog(logger *logrus.Logger, logPath string) error {
 	logFilePath := path.Join(options.HomeDir, "logs", logPath)
-
 	opts := []dflog.Option{
 		dflog.WithLogFile(logFilePath),
 		dflog.WithSign(fmt.Sprintf("%d", os.Getpid())),
-		dflog.WithDebug(options.Debug),
+		dflog.WithDebug(cfg.Debug),
```

Comment:
Could you please make a check for other similar issues just like above?

Reply:
yes